test/ktn_meta_suite_SUITE.erl

```diff
@@ -13,6 +13,12 @@
 
 -type config() :: [{atom(), term()}].
 
+-if(?OTP_RELEASE >= 24).
+-behaviour(ct_suite).
+-else.
+-hank([{unnecessary_function_arguments, [end_per_suite/1]}]).
+-endif.
```

Comment:
Hank does handle ct suites prior to `24` (when `ct_suite` behavior didn't exist).
I guess the issue is that it's only for _some_ variants of `OTP23` (i.e. it was introduced in… say… `OTP23.2` or something).
In any case… if we _really really_ need to preserve the `-hank(…)` part, it should only be there for `?OTP_RELEASE == 23`.

Reply:
I moved it to a simple `if`. As long as CI uses 23.2+ (or whatever) we should be Ok.